### PR TITLE
Separation

### LINKED ISSUE #902: [Feature] Storage computing separation(POC)
**Is your feature request related to a problem? Please describe.**
Resource contention may occur when query engines and storage engines share the same asynchronous runtime

**Describe the solution you'd like**
Storage computing separation

### PR DESCRIPTION
**Pull request by roseboy-liu**
# Which issue does this PR close?

Closes #902 

# Rationale for this change

# What changes are included in this PR?

# Are there any user-facing changes?

### CODE REVIEW

File: main/src/main.rs [hunk 1]
```diff
@@ -174,62 +175,149 @@ fn main() -> Result<(), std::io::Error> {
...
+
+                (
+                    server_builder.build().expect("build server."),
+                    Some(kv_inst),
+                )
             }
-            SubCommand::Run {} => {
+            SubCommand::Query {} => {
                 let meta_manager: MetaRef =
                     RemoteMetaManager::new(global_config.cluster.clone()).await;
```
Comment (bartliu827) [inline main/src/main.rs:234]: remove it?

File: coordinator/src/service.rs [hunk 2]
```diff
@@ -321,7 +318,7 @@ impl CoordService {
 
         let now = tokio::time::Instant::now();
         info!("select statement execute now: {:?}", now);
-        let executor = QueryExecutor::new(option, kv_inst, meta, sender.clone());
+        let executor = QueryExecutor::new(option, kv_inst.clone(), meta, sender.clone());
```
Comment (bartliu827) [inline coordinator/src/service.rs:321]: remove the clone()?

File: main/src/http/http_service.rs [hunk 3]
```diff
@@ -141,6 +165,26 @@ impl HttpService {
...
+            .or(self.metrics())
+            .or(self.print_meta())
+            .or(self.prom_remote_read())
+    }
+
+    fn routes_store(
+        &self,
+    ) -> impl Filter<Extract = (impl warp::Reply,), Error = warp::Rejection> + Clone {
+        self.ping()
+            .or(self.write_line_protocol())
```
Comment (bartliu827) [inline main/src/http/http_service.rs:182]: Write  operation have to distributed routing; driectly write will result to read abnormal.

**Review by bartliu827 — DISMISSED** [review]

File: main/src/http/http_service.rs (hunk 3 above)
Comment (roseboy-liu) [reply main/src/http/http_service.rs:182]: write request routing to tskv write coodinator

File: coordinator/src/service.rs (hunk 2 above)
Comment (roseboy-liu) [reply coordinator/src/service.rs:321]: got

File: main/src/main.rs (hunk 1 above)
Comment (roseboy-liu) [reply main/src/main.rs:234]: got

**Review by bartliu827 — APPROVED** [review]

**Pull request merged**